test/PlayerTest.cpp

```diff
@@ -6,4 +6,77 @@ struct PlayerTest : public ::testing::Test{};
 TEST_F(PlayerTest, vagueValidateInputTest){
     Player p;
     EXPECT_EQ(p.validateInput("xoxoxo"), true);
-}
\ No newline at end of file
+}
+
+TEST_F(PlayerTest, TestName){
+//Given: on input for single player 'adam:'
+    Player p;
+    p.loadData("adam:");
+    EXPECT_EQ(p.getName(),"adam");
+}
+TEST_F(PlayerTest, TestScoreCountOnePoint){
+//Given: on input for single player 'adam:1'
+    Player p;
+    p.loadData("adam:1");
+    EXPECT_EQ(p.getPoints(),1);
+}
+TEST_F(PlayerTest, TestScoreCountOneFrame){
+//Given: on input for single player 'adam:1'
+    Player p;
+    p.loadData("adam:11");
+    EXPECT_EQ(p.getPoints(),2);
+}
+TEST_F(PlayerTest, TestScoreCountTwoFrames){
+//Given: on input for single player 'adam:11|11'
+    Player p;
+    p.loadData("adam:11|11");
+    EXPECT_EQ(p.getPoints(),4);
+}
+TEST_F(PlayerTest, TestScoreCountFrameWithMiss){
+//Given: on input for single player 'adam:11|1-'
+    Player p;
+    p.loadData("adam:11|1-");
+    EXPECT_EQ(p.getPoints(),3);
+}
+TEST_F(PlayerTest, TestScoreCountFrameWithSpare){
+//Given: on input for single player 'adam:11|2/'
+    Player p;

```

Comment:
Player p is in many tests. Please move it to PlayerTest class, so that you will no longer need to provide it in every test.